samples/System.IO.Pipelines.Samples/System.IO.Pipelines.Samples.csproj

```diff
@@ -21,5 +21,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Server.Abstractions" Version="1.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="1.1.0" />
+    <PackageReference Include="System.Net.NameResolution" Version="4.4.0-beta-24913-02" />
```

Comment:
If we foresee any other projects using this assembly, we might also want to move this into the props file. It allows us to manage it with Maestro if / when we need a newer version.